### PR TITLE
Added client meetings from 21, 23

### PR DESCRIPTION
**Pull request by Nick-McGee**
I've added client meetings from 21st and 23rd with Jeff and Yash. Also viewable on Google Docs.

### CODE REVIEW

File: Client Meeting Notes/October 23, 2020 Client Meeting.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+See if that Python library can run our data models. If not, it’ll have to run outside the game.
+
+The problem with the electron app is it is hard to connect to the game, and doesn’t have Python in it -> it doesn’t work with Azure Notebooks.
+
+Yash will give us more details about Minecraft Builtins Python Library.
+
+Know about Encryption with web sockets.
+The game will drop the commands if they aren’t queued, we can’t just send a continuous stream.
+
+Go through the Electron app and understand how it’s working. Another meeting Wednesday to go through it and get a deeper understanding.
```
Comment (Ansper11) [inline Client Meeting Notes/October 23, 2020 Client Meeting.txt:16]: So if Yash adds the package we needed in the built-in python package, then we will not use the Electron app anymore right?

**Comment by Ansper11** [comment]
thanks for taking notes

File: Client Meeting Notes/October 23, 2020 Client Meeting.txt (hunk 1 above)
Comment (Nick-McGee) [reply Client Meeting Notes/October 23, 2020 Client Meeting.txt:16]: That's my understanding, yes

**Comment by yyht6300a** [comment]
Thanks for your work!

**Pull request merged**